test/e2e/actions/steps.go

```diff
@@ -71,6 +71,13 @@ func WaitProject(data *model.TestDataProvider, generation string) {
 	ExpectWithOffset(1, atlasProject.Status.ID).ShouldNot(BeNil(), "Kubernetes resource: Status has field with ProjectID")
 }
 
+func WaitProjectWithoutGenerationCheck(data *model.TestDataProvider) {
+	EventuallyWithOffset(1, kube.GetReadyProjectStatus(data), "15m", "10s").Should(Equal("True"), "Kubernetes resource: Project status `Ready` should be 'True'")
+	atlasProject, err := kube.GetProjectResource(data)
+	Expect(err).ShouldNot(HaveOccurred())
+	ExpectWithOffset(1, atlasProject.Status.ID).ShouldNot(BeNil(), "Kubernetes resource: Status has field with ProjectID")
```

Comment:
The comment here is a little bit misleading as the field names do not match. Not critical thought:)

Reply:
You're talking about line 78, did I understand you correctly?

Reply:
Thanks. Done)